.github/workflows/nightly-bnb.yml

```diff
@@ -82,6 +82,13 @@ jobs:
           pip install -e . --no-deps
           pip install pytest-reportlog pytest-cov parameterized datasets scipy einops
           mkdir transformers-clone && git clone https://github.com/huggingface/transformers.git transformers-clone
+          if [ "${{ matrix.docker-image-name }}" == "huggingface/peft-gpu-bnb-latest:latest" ]; then
+            cd transformers-clone
+            transformers_version=$(pip show transformers | grep '^Version:' | cut -d ' ' -f2 | sed 's/\.dev0//')
```

Comment:
This assumes that there is never going to be a `.dev1`, right? I wonder if something along the lines of `git checkout $(git tag --list --sort=-v:refname | head -n 1)` would be more robust, but there might be a reason to do it this way that's eluding me, so no blocker.

Reply:
Normally, we should not even get a `.dev0` since we are just installing the latest version on pip. As for the tag, I thought about getting the latest tag but I wasn't certain whether the tags had additional uses beyond versioning. Let's see if this holds and,   if we came across issues, we will switch to your command.